actors/common/src/main/java/org/sunbird/learner/util/DataCacheHandler.java

```diff
@@ -66,6 +71,16 @@ private void cacheSystemConfig(Map<String, String> configSettings) {
     }
   }
 
+  private void cacheFrameworkFieldsConfig() {
+    Response response =
```

Comment:
Use SystemSettingClient

Reply:
SystemSettingClientImpl.getSystemSettingByFieldAndKey() it takes ( ActorRef actorRef) as an argument, how to pass it through DataCacheHandler.

Reply:
Framework fields config need not be stored in DataCacheHandler. Instead save it in the user management actor.